charts/scalardb/values.yaml

```diff
@@ -123,6 +123,29 @@ scalardb:
     # -- The log level of Scalar DB
     dbLogLevel: INFO
 
+  # -- The database.properties is created based on the values of scalardb.storageConfiguration by default. If you want to customize database.properties, you can override this value with your database.properties.
+  databaseProperties: |
```

Comment:
This is a main point of this PR.
In this PR, we define database.properties as value of values.yaml.
(In other words, we don't need to create `ConfigMap` manually.)

If we use the default value of `scalardb.databaseProperties`, each value of `scalardb.storageConfiguration` applied to `scalardb.databaseProperties` via environment variables.
(The templating will be done by `dockerize` in the container side.)

Also, if we override this value in a custom values file, the `ConfigMap` is created automatically, and we can use the overridden (customized) database.properties.